module/zfs/vdev_removal.c

```diff
@@ -2162,6 +2230,10 @@ spa_removal_get_stats(spa_t *spa, pool_removal_stat_t *prs)
 }
 
 #if defined(_KERNEL)
+module_param(zfs_removal_ignore_errors, int, 0644);
+MODULE_PARM_DESC(zfs_removal_ignore_errors,
+	"Ignore hard IO errors when removing device");
```

Comment:
Maybe say "Ignore hard IO errors when doing a zpool remove", so that it's not confused with a physical device removal.